### PR TITLE
gitlab - display number processes avaliable for tests

### PR DESCRIPTION
**Pull request by jeremylt**
I added this so we can see a bit more info.

### CODE REVIEW

**Comment by jeremylt** [comment]
It looks like the default value for `nproc` in our CI is 64 - is this what we want?

**Comment by jeremylt** [comment]
Ok, now this PR lets us know how many processes we're using while testing and reduces the number of processes for the GPU tests.

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -9,6 +9,7 @@ noether-rocm:
   script:
 # Compilers
     - export COVERAGE=1 CC=gcc CXX=g++ FC=gfortran HIPCC=hipcc
+    - echo "---------- nproc-----------" && NPROC_CPU=$(nproc) && NPROC_GPU=8
```
Comment (jedbrown) [inline .gitlab-ci.yml:12]: Should `NPROC_GPU` be a minimum of `$(nproc)` and some static (or determined) maximum?

**Review by jedbrown — APPROVED** [review]
I'm okay with this if it's needed, though I want there to be a better way, such a resource manager that limits the number of concurrent jobs on CPU and GPU. https://github.com/flux-framework/flux-core/pull/3567#issuecomment-808372257

File: .gitlab-ci.yml (hunk 1 above)
Comment (jeremylt) [reply .gitlab-ci.yml]: agreed and updated

**Comment by jeremylt** [comment]
I feel like this is a good stopgap for now and we can make an issue to remind us to move to something more sophisticated in the future

**Review by jedbrown — APPROVED** [review]
Thanks, looks good to squash-merge.

**Pull request merged**